lib/t/accounts.ex

```diff
@@ -532,6 +532,25 @@ defmodule T.Accounts do
     Media.user_s3_url()
   end
 
+  def voice_upload_form(content_type) do
+    Media.sign_form_upload(
+      key: Ecto.UUID.generate(),
+      content_type: content_type,
+      # 50 MB
+      max_file_size: 50_000_000,
+      expires_in: :timer.hours(1),
+      acl: "private"
+    )
+  end
+
+  def voice_s3_url do
+    Media.user_s3_url()
+  end
+
+  def voice_url(s3_key) do
+    Media.user_presigned_url(s3_key)
```

Comment:
I see it's presigned, but it probably doesn't have to be since the asset is "public".